src/vss_tools/vspec/tree.py

```diff
@@ -271,7 +271,7 @@ def delete_nodes(self, nodes: tuple[VSSNode]) -> None:
             node.parent = None
         size_after = self.size
         if nodes:
-            log.info(f"Nodes deleted, given={len(nodes)}, overall={size_before - size_after}")
+            log.debug(f"Nodes deleted, given={len(nodes)}, overall={size_before - size_after}")
```

Comment:
When this was in INFO mode, my console got flooded with below message:

`INFO     Nodes deleted, given=1, overall=0`

which on my opinion still does not say much i.e., which node got deleted from where.

By the way, I am calling the deletion, only once in the **filter_vss_tree** function. 
Actually, the function is calling itself recursively for each child node, so the **delete_nodes** gets called on each execution of **filter_vss_tree**.
Anyway, I took it outside of the **filter_vss_tree** function, and called the **delete_nodes** in the vss2samm.py, rjust after the **filter_vss_tree** function completed with marking of nodes for deletion, and still got the same output in the console. Exactly the same as above INFO message "flooded" my console again.

Anyway, as you want to see it, then I will rollback this change.

Reply:
if overall = 0 then something is odd since you should only delete nodes that are reachable from the node you are calling the delete.

I agree that when calling this more than  once that the output is not good or helpful.
Right now it was only called once: for the vss tree that has been loaded

Reply:
Thank you, I got it :)

After I reworked the deletion and move the call to delete, outside the filter function, I got just 1 message stating just deleted nodes which was more than enough :)